syntax = "proto3";

package happystock.happystockapi;

import "google/protobuf/timestamp.proto";

message stockSentiment {
    string name =1;
    google.protobuf.Timestamp date = 2;
    int32 sentiment = 3;
}

message stockPrice {
    string name =1;
    google.protobuf.Timestamp date = 2;
    double price =3;
}

message listStockSentiment {
    repeated stockSentiment sentimentList = 1;
    string tweetExample = 2;
    int32 tweetSentiment = 3;
}

message listStockPrice{
    repeated stockPrice priceList = 1;
}